build.gradle

```diff
@@ -315,8 +315,8 @@ signing {
 
 
 //signing {
-//    def signingKey = findProject("signingKey")
```

Comment:
I think I'll bring back the old one and leave both options commented out, since we use a regular old key for signing (I think).